config/routes.rb

```diff
@@ -10,6 +10,7 @@
       delete 'remove_dev', to: "projects#remove_dev"
       get 'payment', to: "projects#submit_payment"
     end
+    patch '/deactivate', to: "contractors#deactivate"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.